app/person-escort-record/controllers/framework-step.test.js

```diff
@@ -503,19 +505,12 @@ describe('Person Escort Record controllers', function () {
         })
 
         it('should call correct number of updates', function () {
```

Comment:
The wording of this test should probably be changed, as its no longer checking how many updates have occurred.